examples/step-31/step-31.cc

```diff
@@ -1124,12 +1124,9 @@ namespace Step31
 
     Amg_preconditioner = std::make_shared<TrilinosWrappers::PreconditionAMG>();
 
-    std::vector<std::vector<bool>>   constant_modes;
     const FEValuesExtractors::Vector velocity_components(0);
-    DoFTools::extract_constant_modes(stokes_dof_handler,
-                                     stokes_fe.component_mask(
-                                       velocity_components),
-                                     constant_modes);
+    const auto constant_modes = DoFTools::extract_constant_modes(
```

Comment:
We try to avoid `auto` in the tutorial programs. (Ask @jppelteret for why :-) ) Just spell out the returned object type.